cumulus/parachains/runtimes/collectives/collectives-westend/src/fellowship/mod.rs

```diff
@@ -0,0 +1,238 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+//! The Westend Technical Fellowship.
```

Comment:
Why is this added? Should we not use a bridge instead of mirror everything?

Reply:
I think in this case we want it for testing purpose. Also it is handy to try and test your frame crates within polkado-sdk, before it get published and can be applied in fellows runtime repo

Reply:
It was here from Polkadot Collectives because deployed it on Westend first. I still think having some Fellowship directly on the testnet is useful. For example, I just added Polkassembly to it on Westend so they could test UI flows for Fellows. Maybe we should reduce the members for people who don't want to be on Westend as well.

Reply:
Okay, I guess its not the `Technical Fellowship` then but just some test fellowship?  
Maybe some parameters with faster turnouts and so on that it is easier to test UI. You could also invite the Polkassembly guys there directly.

Reply:
IIRC we did add some faster parameters for it.